telegram/message.py

```diff
@@ -301,6 +305,31 @@ def de_json(cls, data, bot):
 
         return cls(bot=bot, **data)
 
+    @property
+    def effective_attachment(self):
+        """
+        Optional[Union[:class:`telegram.Audio`, :class:`telegram.Game`, :class:`telegram.Document`,
```

Comment:
Shouldn't it be:
```
Returns:
    :class:`telegram.Audio` | :class:`telegram.Game` | :class:`telegram.Document` [...]
```
I'm not sure our sphinx parser understands the optional and union very well